### PR TITLE
[Fix](HttpServer) Refactor API Endpoints to Only Allow GET Requests for Enhanced Security

### PR DESCRIPTION
**Pull request by zy-kkk**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by CalvinKirs — APPROVED** [review]

File: fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/MetricsAction.java [hunk 1]
```diff
@@ -41,7 +42,7 @@ public class MetricsAction extends RestBaseController {
     private static final Logger LOG = LogManager.getLogger(MetricsAction.class);
     private static final String TYPE_PARAM = "type";
 
-    @RequestMapping(path = "/metrics")
+    @RequestMapping(path = "/metrics", method = RequestMethod.GET)
```
Comment (CalvinKirs) [inline fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/MetricsAction.java:45]: Useing `@GetMapping`？

**Comment by zy-kkk** [comment]
run buildall

**Comment by zy-kkk** [comment]
run buildall

**Comment by zy-kkk** [comment]
run buildall

**Comment by zy-kkk** [comment]
run buildall

**Comment by zy-kkk** [comment]
run buildall

**Comment by zy-kkk** [comment]
run buildall

**Comment by zy-kkk** [comment]
run feut

**Comment by zy-kkk** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**